plexapi/library.py

```diff
@@ -555,10 +555,9 @@ def search(self, title=None, sort=None, maxresults=None,
                     album, track; optional).
                 container_start (int): default 0
                 container_size (int): default X_PLEX_CONTAINER_SIZE in your config file.
-                custom_filter_string (str): A custom attribute string to append to the API call IE: &somefield!=value
                 **kwargs (dict): Any of the available filters for the current library section. Partial string
-                        matches allowed. Multiple matches OR together. Negative filtering also possible, just add an
```

Comment:
Reading this documentation it basically states that `updatedAt>=XYZ&lastViewedAt>=XYZ` should result in all items being returned which where either watched or updated since `XYZ`. But IIRC you stated that it doesn't work. Did you try it?

Reply:
I didn't try it using their library, but tried it with direct API calls and it AND'd them together not OR'd them.

Also in [these docs](https://github.com/Arcanemagus/plex-api/wiki/Plex-Web-API-Overview) of the api it calls out `When multiple parameters are passed, they are logically ANDed together`